README.md

```diff
@@ -70,22 +70,19 @@ Testground aims to be tightly integrated with the software engineering practices
 
 The current Testground Team is composed of:
 
-- @raulk - Lead Architect, Engineer, Developer
-- @daviddias - Engineer, Developer, acting as interim PM for the project
-- @nonsense - Engineer, Developer, Testground as a Service / Infrastructure Lead
-- @jimpick - Engineer, Developer
-- @stebalien - Engineer, Developer
-- @hacdias - Engineer, Developer
+- @raulk - Architect, Lead Software Engineer
+- @nonsense - Software Engineer, Testground as a Service / Infrastructure Lead
+- @fabiomartins91 - [Technical Project Manager (TPM)](https://github.com/ipfs/team-mgmt/blob/master/TEAMS_ROLES_STRUCTURES.md#working-group-technical-project-manager-tpm)
+- @hacdias - Software Engineer
+- @daviddias - ?? Figuring it out :D
```

Comment:
Yeah. I'll tag myself as soft eng for now. I've updated it so that @FabioMartins91 gets to know who is on the team :)

Reply:
@daviddias It would be great if the name of the branch was more informative. It's easier for new people to understand if the name of the branch gives hints of the work that was done ;) thanks